docs/source/utils.rst

```diff
@@ -1,3 +1,14 @@
+utils package
+=============
+
+utils.experiment_options.experimentoptions module
+-------------------------------------------------
+
+.. automodule:: utils.experiment_options.experimentoptions
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
 =================
 Config Parameters
 =================
```

Comment:
On the TODO: tx_subdev, main_rx_subdev?? TODO: inteferometer_rx_subdev?? line, it is too long so the table explodes out too wide. Can you find a way to make this not happen?

Similarly, with the large description for minimum_pulse_separation, the table is really wide again.